dist/index.js

```diff
@@ -233,7 +236,10 @@ var TsType;
                 //Multiple line description in comment above
                 if (settings.propertyDescription && _.type.description && !_.type.id)
                     decl = (_.type.description.search(newLineRegex) > 0) ?
-                        _.type.description.split(newLineRegex).map(function (line) { return indentString + '// ' + line; }).join('\n') + '\n' + decl :
+                        _.type.description
+                            .split(newLineRegex)
+                            .map(function (line, lineNum) { return indentString + (lineNum > 0 ? multiLineCommentIndent : multiLineCommentStart) + line; })
```

Comment:
As you know, `index.js` is generated by `gulp dist`. Curiously, it is declaring `function (line, lineNum)` rather than just `(line, lineNum)`...